app/controllers/main.js

```diff
@@ -1,52 +0,0 @@
-'use strict';
```

Comment:
Could you please make 2 commits:
 - First moving it to the new place
 - Second modifying the file

This makes it difficult to find the diff of what you've changed as it just shows a glob has been deleted and nother added.